be/src/column/const_column.cpp

```diff
@@ -12,10 +13,17 @@ namespace starrocks::vectorized {
 
 ConstColumn::ConstColumn(ColumnPtr data) : _data(std::move(data)), _size(0) {
     DCHECK(!_data->is_constant());
+    // _data->resize(1);
+    // DCHECK(_data->size() == 1);
 }
 
 ConstColumn::ConstColumn(ColumnPtr data, size_t size) : _data(std::move(data)), _size(size) {
     DCHECK(!_data->is_constant());
+    std::cout << _data->size() << std::endl;
+    if (_data->size() > 1) {
+        _data->resize(1);
+        DCHECK(_data->size() == 1);
```

Comment:
the DCHECK is useless, because the code above already resize it to 1

Reply:
sorry, i submit a error version without some modify, i'll push a new version after unittest,  wait a moment